func main() int {
	return f(13)
}

func f(n int) int {
	if n <= 1 {
		return n
	}

	return f(n - 1) + f(n - 2)
}
======
stdout:
stderr:
exit status: 233
